autoconfigure/collector-eventhub/src/test/java/zipkin/autoconfigure/collector/eventhub/EventHubCollectorAutoConfigurationTest.java

```diff
@@ -31,184 +31,181 @@
-
-        String storageBlobPrefix = "pashmak";
-
-        context = new AnnotationConfigApplicationContext();
-        addEnvironment(context, "zipkin.collector.eventhub.storageConnectionString:" + dummyEventHubConnectionString);
-        addEnvironment(context, "zipkin.collector.eventhub.eventHubConnectionString:" + dummyStorageConnectionString );
-        addEnvironment(context, "zipkin.collector.eventhub.storageBlobPrefix:" + storageBlobPrefix);
-        context.register(PropertyPlaceholderAutoConfiguration.class,
-                EventHubCollectorAutoConfiguration.class,
-                InMemoryConfiguration.class);
-        context.refresh();
-
-        EventHubCollector collector = context.getBean(EventHubCollector.class);
-        assertNotNull(collector);
-        assertEquals(storageBlobPrefix, collector.getStorageBlobPrefix());
-    }
-
-    @Test
-    public void provideCollectorComponent_canSetStorageContainerName() {
-
-        String storageContainerName = "pashmak";
-
-        context = new AnnotationConfigApplicationContext();
-        addEnvironment(context, "zipkin.collector.eventhub.storageConnectionString:" + dummyEventHubConnectionString);
-        addEnvironment(context, "zipkin.collector.eventhub.eventHubConnectionString:" + dummyStorageConnectionString );
-        addEnvironment(context, "zipkin.collector.eventhub.storageContainerName:" + storageContainerName);
-        context.register(PropertyPlaceholderAutoConfiguration.class,
-                EventHubCollectorAutoConfiguration.class,
-                InMemoryConfiguration.class);
-        context.refresh();
-
-        EventHubCollector collector = context.getBean(EventHubCollector.class);
-        assertNotNull(collector);
-        assertEquals(storageContainerName, collector.getStorageContainerName());
+  AnnotationConfigApplicationContext context;
+
+  static final String dummyEventHubConnectionString = "endpoint=sb://someurl.net;SharedAccessKeyName=dumbo;SharedAccessKey=uius7y8ewychsih";
+  static final String dummyStorageConnectionString = "a=b;c=d";
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+
+  @After
+  public void close() {
+    if (context != null) context.close();
+  }
+
+  @Test
+  public void doesntProvideCollectorComponent_whenEventHubConnectionStringUnset() {
+    context = new AnnotationConfigApplicationContext();
+    context.register(PropertyPlaceholderAutoConfiguration.class,
+        EventHubCollectorProperties.class,
+        EventHubCollectorAutoConfiguration.class,
+        InMemoryConfiguration.class);
+    context.refresh();
+
+    thrown.expect(NoSuchBeanDefinitionException.class);
+    context.getBean(EventHubCollector.class);
+  }
+
+  @Test
+  public void provideCollectorComponent_whenEventHubConnectionStringIsSet() {
+    context = new AnnotationConfigApplicationContext();
+    addEnvironment(context, "zipkin.collector.eventhub.storageConnectionString:" + dummyStorageConnectionString);
```

Comment:
these properties are quite long compared to what's normally done in zipkin. I think we should shorten them.

For example, the zookeeper connection string for kafka is "zipkin.collector.kafka.zookeeper" which ends up put as an env variable KAFKA_ZOOKEEPER

short names like this lead to a much better ops experience than say KAFKA_ZOOKEEPER_CONNECTION_STRING, not only reducing the typo chances, but also making it easy to write recipes, one-line startup scripts etc.

We took a while creating this part of the readme, maybe we can do a "README first" development here, copying that formatting and abbreviation style 

https://github.com/openzipkin/zipkin/blob/master/zipkin-server/README.md#kafka-collector